File: LeetCode/Easy/Jewels and Stones/771.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        Set<Character> set = new HashSet<>();
        for (char jewel : jewels.toCharArray()) set.add(jewel);

        int count = 0;
        for (char stone : stones.toCharArray()) count += (set.contains(stone)) ? 1 : 0;

        return count;
    }
}

File: LeetCode/Easy/Jewels and Stones/771.js
/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function(jewels, stones) {
    const set = new Set(jewels);
    const count = [...stones].reduce((acc, cur) => set.has(cur) ? acc + 1 : acc, 0);
    return count;
};
